dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -30,9 +30,24 @@ max=Maximum concurrency
 max.desc=Specifies the maximum number of tasks that can run simultaneously. The default is Integer.MAX_VALUE. Maximum concurrency can be updated while tasks are in progress. If the maximum concurrency is reduced below the number of concurrently running tasks, the update goes into effect gradually as in-progress tasks complete, rather than canceling them.
 
 maxPolicy=Maximum policy
-maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum concurrency for tasks that run on the task submitter's thread. Tasks can run on the task submitter's thread when using the untimed invokeAll method, or, if only invoking a single task, the untimed invokeAny method. If the run-if-queue-full attribute is configured, it is also possible for tasks to run the task submitter's thread when using the execute and submit methods. In all of these cases, this attribute determines whether or not running on the submitter's thread counts against the maximum concurrency.
-maxPolicy.loose.desc=Maximum concurrency is loosely enforced. Tasks are allowed to run on the task submitter's thread without counting against maximum concurrency.
-maxPolicy.strict.desc=Maximum concurrency is strictly enforced. Tasks that run on the task submitter's thread count towards maximum concurrency. This policy does not allow tasks to run on the task submitter's thread when already at maximum concurrency.
+maxPolicy.desc=Indicates whether to loosely or strictly enforce maximum \
+ concurrency for tasks that run on the task submitter's thread. If the \
```

Comment:
I recommend breaking this sentence up so it's not so long. Itt's also best to avoid elliptical clauses like "when using" and "if only invoking"- the subject is not clear (ie what's doing the  using/invoking) and that makes the sentence difficult to translate. I'm not sure what to suggest because Im not sure what the intended subject is. 

For example, if the subject is "policy", I'd recommend something like

If the concurrency policy is not configured to run tasks on virtual threads, tasks can \
 run on the task submitter's thread when the policy uses the untimed invokeAll method.  \
 If the policy is invoking only a single task, the task can run on the submitter's thread when the policy uses the untimed invokeAny method. \
 
that's probably incorrect but you get the idea. If it is the developer who is using/invoking, you can write directly to them in 2nd person, ie "when you use the untimed invokeAll method"

Reply:
Thanks for reviewing. I added your other suggestions, and addressed the above in https://github.com/OpenLiberty/open-liberty/pull/29787/commits/4516a85ca6e23e937040eebc9eb9c4978a32c99d